armotypes/portaltypes.go

```diff
@@ -145,6 +145,12 @@ type PortalCluster struct {
 	LastLoginDate    string `json:"last_login_date,omitempty" bson:"last_login_date,omitempty"`
 }
 
+type StripeCustomer struct {
+	CustomerID         string `json:"customer_id,omitempty" bson:"customer_id,omitempty"`
+	SubscriptionID     string `json:"subscription_id,omitempty" bson:"subscription_id,omitempty"`
+	SubscriptionStatus string `json:"subscription_status,omitempty" bson:"subscription_status,omitempty"`
```

Comment:
please add the possible statuses as a comment

Reply:
done